src/lib.rs

```diff
@@ -284,6 +284,74 @@ impl<T> ConcurrentQueue<T> {
         }
     }
 
+    /// Attempts to return the value stored at the head of the queue.
+    /// 
+    /// If the queue is empty, an error is returned.
+    /// 
+    /// # Examples
+    /// 
+    /// ```
+    /// use concurrent_queue::{ConcurrentQueue, GetError};
+    /// 
+    /// let mut q = ConcurrentQueue::bounded(2);
```

Comment:
I'm specifically using size 2 because I haven't implemented the methods for the `Single` type. But I intend to change this so we can have a clear pattern across all tests.